### ISSUE TITLE
AVT 2 DatePicker date field placeholder fails contrast for regular text

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4231]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
Color Contrast Analyzer

## Detailed Description
Go to the React Datepicker Component
Run Color Contrast Analyzer and test the search placeholder text against the background color (see screenshot):
<img width="1248" alt="Screen Shot 2019-10-04 at 1 41 47 PM" src="https://user-images.githubusercontent.com/21676914/66231756-dee7b000-e6ac-11e9-8196-e56644b4a303.png">
